stl/inc/xstring

```diff
@@ -2569,37 +2547,43 @@ private:
             return;
         }
 
-        // We need to check whether we have a misaligned SSO buffer because of the proxy in `_Container_base` (e.g. x86)
+        // When SSO is active, the buffer may not be aligned on an ASan shadow memory boundary;

```

Comment:
Could we say "might not" to avoid the English language's confusion over whether "may not" means "might not" or "must not"?

Reply:
Our variable names are all `_Asan_meow`, but this says "ASan". Can we pick a capitalization and use it consistently everywhere?

Reply:
I don't love this suggestion; `ASan` is the more "correct" capitalization as a shortening of "AddressSanitizer", but for variable names, `asan` is more appropriate. At the beginning of a name, due to our naming style, it should be `_Asan`.

Reply:
That's a reasonable argument - I withdraw my naming gripe. My concern about "may not" remains.

Reply:
Yeah, working on fixing.